superset/connectors/sqla/models.py

```diff
@@ -67,14 +68,16 @@
 from superset.models.helpers import AuditMixinNullable, QueryResult
 from superset.result_set import SupersetResultSet
 from superset.sql_parse import ParsedQuery
-from superset.typing import Metric, QueryObjectDict
+from superset.typing import AdhocMetric, Metric, OrderBy, QueryObjectDict
 from superset.utils import core as utils
 from superset.utils.core import GenericDataType
 
 config = app.config
 metadata = Model.metadata  # pylint: disable=no-member
 logger = logging.getLogger(__name__)
 
+VIRTUAL_TABLE_ALIAS = "expr_qry"
```

Comment:
I've always thought "expression query" sounds funny, as this is really just a "subquery". However, since we have other subqueries as well, maybe that's not the best name. "virtual_table" could be ok, as we use that term throughout the application.